Sources/Interfaces.h

```diff
@@ -104,6 +105,7 @@ class IGame
     virtual TextPainter& GetTexts() = 0;
     virtual SyncRandom& GetRandom() = 0;
     virtual Names& GetNames() = 0;
+    virtual IGame* GetTrueGame() = 0;
 
```

Comment:
You should avoid to make changes in any sources dirs except /core/objects/. Changes in other folders requests fully understanding of source code and they will be changes of the engine (and such changes should be made really carefully).